src/components/ProductCreateForm/ProductCreateForm.tsx

```diff
@@ -78,15 +80,39 @@ export const ProductCreateForm = (): ReactElement => {
   };
 
   const handleServerErrors = (problems: productProblemDetails): void => {
+    const errorList = [];
+
+    errorList.push(
+      <div>
+        {problems.title}
+        <br />
+        <br />
+      </div>
+    );
+
     if (problems.errors.name) {
-      toast.error(problems.errors.name[0]);
+      errorList.push(
+        <li>
+          {problems.errors.name[0]} <br />
+        </li>
+      );
     }
     if (problems.errors.price) {
-      toast.error(problems.errors.price[0]);
+      errorList.push(
+        <li>
+          {problems.errors.price[0]} <br />
+        </li>
+      );
     }
     if (problems.errors.description) {
-      toast.error(problems.errors.description[0]);
+      errorList.push(
+        <li>
+          {problems.errors.description[0]} <br />
+        </li>
+      );
     }
+
+    toastId.current = toast.error(<ul>{errorList}</ul>) as number;
```

Comment:
imo error toasts shouldn't autoclose at all, or at least autoclose after like 10 or 15 seconds. It can take a bit of time to read all the errors if there are multiple 

I think you can adjust this by passing in a config object to `toast.error()`

Reply:
Changed that. It will now not auto close. Note it will stay open even if you navigate to another page. We'll resolve this later after UX cos this will be simpler if we use a modal (Kash Ok'd)

Reply:
sounds good 👍